usaspending_api/common/etl/spark.py

```diff
@@ -292,7 +294,8 @@ def load_delta_table(
             If left False (the default), the DataFrame data will be appended to existing data.
     Returns: None
     """
-    logger = get_jvm_logger(spark)
+    logging.basicConfig(level=logging.INFO)
```

Comment:
You also don't need to set the logging level because it is configured as part of the Django settings (https://github.com/fedspendingtransparency/usaspending-api/blob/9add4f7fc6398d2c8dfd46d0dbd144cfbd4fad80/usaspending_api/settings.py#L379).